src/frontend/sharedComponents/SelectOne.tsx

```diff
@@ -26,16 +31,35 @@ export const SelectOne = <T,>({
             : index
         }
         testID={`${item.value}LanguageButton`}
-        onPress={() => value !== item.value && onChange(item.value)}>
+        onPress={() => value !== item.value && onChange(item.value)}
+        style={[radioButtonPosition === 'right' && styles.rowReverse]}>
```

Comment:
nit: to avoid an unnecessary array allocation

```suggestion
        style={radioButtonPosition === 'right' ? styles.rowReverse : undefined}>
```

Reply:
Yeah I don't know why I put that in an array. Sorry. Good catch!